pkg/system/phase4_configuring.go

```diff
@@ -469,8 +469,8 @@ func (r *Reconciler) ReconcileDefaultBackingStore() error {
 		if err := r.prepareGCPBackingStore(); err != nil {
 			return err
 		}
-	} else if r.IsIBMCloud {
-		log.Infof("Running in IBM Cloud. Creating default backing store on IBM objectstore")
+	} else if r.IBMCloudCOSCreds.UID != "" {
+		log.Infof("Running in IBM Cloud. CredentialsRequest %q created. Creating default backing store on IBM objectstore", r.IBMCloudCOSCreds.Name)
```

Comment:
I wouldn't phrase that as a "CredentialsRequest", since the cloud credentials operator doesn't support IBM and you are just reading from the secret that doesn't have to do anything with them (it is created by the IBM deployment flow scripts).
What do you think about:
```log.Infof("IBM objectstore credentials %q created. Creating default backing store on IBM objectstore", r.IBMCloudCOSCreds.Name)```?

Reply:
Agree. will make the change